embrace-android-sdk/src/test/java/io/embrace/android/embracesdk/ndk/EmbraceNdkServiceTest.kt

```diff
@@ -499,13 +512,12 @@ internal class EmbraceNdkServiceTest {
 
         appFramework = Embrace.AppFramework.UNITY
         initializeService()
-        val mockedNdkService = spyk(embraceNdkService, recordPrivateCalls = true)
```

Comment:
The previous test is a bit problematic as the invocation could be on either the mock or the actual object the mock wraps. Changing the only reference to the mock removes that potential gap.